examples/c/README.md

```diff
@@ -37,21 +37,21 @@ Change into the `onnxruntime-genai\examples\c` folder.
 1. Install onnxruntime
    
    ```cmd
-   curl -L https://github.com/microsoft/onnxruntime/releases/download/v1.19.2/onnxruntime-win-x64-1.19.2.zip -o onnxruntime-win-x64-1.19.2.zip
-   tar xvf onnxruntime-win-x64-1.19.2.zip
-   copy onnxruntime-win-x64-1.19.2\include\* include
-   copy onnxruntime-win-x64-1.19.2\lib\* lib
+   curl -L https://github.com/microsoft/onnxruntime/releases/download/v1.20.1/onnxruntime-win-x64-1.20.1.zip -o onnxruntime-win-x64-1.20.1.zip
+   tar xvf onnxruntime-win-x64-1.20.1.zip
+   copy onnxruntime-win-x64-1.20.1\include\* include
+   copy onnxruntime-win-x64-1.20.1\lib\* lib
```

Comment:
Is there something we can do to avoid updating this every release?

Reply:
I didn't even notice the amount of manually scripting needed until now. I think maybe we can reuse or copy some of the code in [ortlib.cmake](https://github.com/microsoft/onnxruntime-genai/blob/main/cmake/ortlib.cmake) and put it in the `CMakeLists.txt` of C example to simplify the process.

Reply:
Sounds good. Shall we do it in a separate PR?